From fa8cc87ce9321e2b8847750994c906cf00ff6ac4 Mon Sep 17 00:00:00 2001
From: Victor Gu <xigu@marvell.com>
Date: Fri, 17 Nov 2017 12:28:43 +0800
Subject: [PATCH 2110/2241] fix: mvebu: dts: a70x0: replace CP i2c1 with MSS
 i2c0 for AMC board

In Armada7040 AMC board, the CP MSS i2c0 is used in hardware
instead of normal CP i2c1 to control the SFP module.
Previously the CP i2c1 is used by mistake which leads to the
failure when access the SFP module by i2c.
This patch fixes it by using the right i2c controller.

Change-Id: I5ff34af00c34ef8378c378df66332610882f2eab
Signed-off-by: Victor Gu <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/46436
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-7020-amc.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7020-amc.dts b/arch/arm64/boot/dts/marvell/armada-7020-amc.dts
index 761cea9..8bc0361 100644
--- a/arch/arm64/boot/dts/marvell/armada-7020-amc.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7020-amc.dts
@@ -111,7 +111,7 @@
 				clock-frequency = <100000>;
 			};
 
-			cp0_i2c1: i2c@701100 {
+			cp0_mss_i2c0: i2c@211000 {
 				status = "okay";
 				clock-frequency = <100000>;
 			};
-- 
2.7.4

